cves/CVE-2013-2115.yml

```diff
@@ -6,33 +7,33 @@ nickname_instructions: |
   Nickname is optional. Provide a useful, professional, and catchy nickname for
   this vulnerability. Ideally fewer than 30 characters. This will be shown
   alongside its CVE to make it more easily distinguished from the rest.
-nickname:
+nickname: OGNL Reflected XSS
 
 CWE_instructions: |
   Please go to cwe.mitre.org and find the most specific, appropriate CWE entry
   that describes your vulnerability. (Tip: this may not be a good one to start
   with - spend time understanding this vulnerability before making your choice!)
-CWE:
+CWE: 79
```

Comment:
Why did you choose this CWE versus others such as 88?

Reply:
I chose 79 because it was accessible directly to the user.

Reply:
https://nvd.nist.gov/vuln/detail/CVE-2013-2115

If you look at the vulnerability type under the technical details section, NIST classifies this as CWE-94